pkg/controller/backup_controller.go

```diff
@@ -792,7 +792,7 @@ func (b *backupReconciler) runBackup(backup *pkgbackup.Request) error {
 		}
 	}
 
-	b.logger.WithField(Backup, kubeutil.NamespaceAndName(backup)).Info("Backup completed")
+	b.logger.WithField(Backup, kubeutil.NamespaceAndName(backup)).Info(fmt.Sprintf("Initial backup processing complete, moving to %s", backup.Status.Phase))
```

Comment:
```suggestion
	b.logger.WithField(Backup, kubeutil.NamespaceAndName(backup)).Info(fmt.Sprintf("Initial backup processing completed, moving to %s", backup.Status.Phase))
```

Reply:
I think "complete" is better. i.e. "processing is complete" -- it avoids confusion with the "Completed" phase.